services/app-web/src/components/ActionButton/ActionButton.tsx

```diff
@@ -81,10 +88,27 @@ export const ActionButton = ({
                 : inheritedProps['aria-label'],
     }
 
+    const onClickHandler = (
+        e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+    ) => {
+        logButtonEvent({
+            text: extractText(children),
+            button_style: variant,
```

Comment:
For saving time at this stage and to reduce back and forth with Blast team, I would like to just have us use the specs Blast has provided when possible. That means here can we map all `outline` variant to be `transparent` please? e.g. something like  `variant === 'outline'? 'transparent' : variant`